corebehrt/classes/heads.py

```diff
@@ -2,16 +2,16 @@
 
 
 class MLMHead(torch.nn.Module):
-    def __init__(self, config):
+    def __init__(self, hidden_size: int, vocab_size: int, layer_norm_eps: float):
         super().__init__()
         # BertPredictionHeadTransform
-        self.dense = torch.nn.Linear(config.hidden_size, config.hidden_size)
+        self.dense = torch.nn.Linear(hidden_size, hidden_size)
         self.activation = torch.nn.GELU()
-        self.LayerNorm = torch.nn.LayerNorm(config.hidden_size, eps=config.layer_norm_eps)
+        self.LayerNorm = torch.nn.LayerNorm(hidden_size, eps=layer_norm_eps)
```

Comment:
Should probably keep this to snake_case instead of CamelCase (I can see github "sees" it as a Class instead of an attribute)